packages/react-jss/src/createHoc.js

```diff
@@ -247,21 +247,24 @@ export default function createHOC<
     }
   }
 
-  return function JssContextSubscriber(props) {
-    return (
-      <JssContext.Consumer>
-        {context => {
-          if (isThemingEnabled || injectTheme) {
-            return (
-              <ThemeConsumer>
-                {theme => <Jss theme={theme} {...props} jssContext={context} />}
-              </ThemeConsumer>
-            )
-          }
-
-          return <Jss {...props} jssContext={context} />
-        }}
-      </JssContext.Consumer>
-    )
-  }
+  // $FlowFixMe: Sadly there is no support for forwardRef yet
+  const JssContextSubscriber = React.forwardRef((props, ref) => (
+    <JssContext.Consumer>
+      {context => {
+        if (isThemingEnabled || injectTheme) {
+          return (
+            <ThemeConsumer>
+              {theme => <Jss innerRef={ref} theme={theme} {...props} jssContext={context} />}
```

Comment:
do we still need innerRef here? or can it be dirrectly ref?

Reply:
We still need `innerRef` because we don't want the user to have the instance of the HOC.